env/index.js

```diff
@@ -102,6 +102,18 @@ class Env {
 
     return env === 'testing'
   }
+
+  /**
+   * Determines whether the `NODE_ENV` environment
+   * variable is set to the given `environment`.
+   *
+   * @returns {Boolean}
+   */
+
```

Comment:
Can you please remove this empty line between the docblock and function?

Reply:
Done :)